torch_xla/debug/metrics.py

```diff
@@ -79,3 +79,8 @@ def short_metrics_report(counter_names: list = None, metric_names: list = None):
         'TransferToDeviceTime', 'TransferFromDeviceTime'
     ]
   return torch_xla._XLAC._short_xla_metrics_report(counter_names, metric_names)
+
+
+def executed_fallback_ops():
```

Comment:
cc @alanwaketan we can start using this new api which is better than do a string assertIn from the metrics_report,